src/main/java/org/wso2/carbon/module/swiftiso20022/JsonToMT940Transformer.java

```diff
@@ -20,22 +20,20 @@
 
 import com.google.gson.Gson;
 import org.apache.axis2.AxisFault;
-import org.apache.commons.lang3.StringUtils;
 import org.apache.synapse.MessageContext;
 import org.apache.synapse.core.axis2.Axis2MessageContext;
 import org.json.JSONException;
 import org.json.JSONObject;
 import org.wso2.carbon.connector.core.AbstractConnector;
 import org.wso2.carbon.connector.core.ConnectException;
 import org.wso2.carbon.module.swiftiso20022.constants.ConnectorConstants;
-import org.wso2.carbon.module.swiftiso20022.model.ErrorModel;
-import org.wso2.carbon.module.swiftiso20022.mt940models.RequestPayloadModel;
-import org.wso2.carbon.module.swiftiso20022.mt940models.TransactionModel;
+import org.wso2.carbon.module.swiftiso20022.constants.MT940Constants;
+import org.wso2.carbon.module.swiftiso20022.models.mt940models.RequestPayloadModel;
 import org.wso2.carbon.module.swiftiso20022.utils.ConnectorUtils;
 import org.wso2.carbon.module.swiftiso20022.utils.JsonToMt940Utils;
-import org.wso2.carbon.module.swiftiso20022.utils.ValidatorUtils;
+import org.wso2.carbon.module.swiftiso20022.validation.JsonToMT940PayloadValidator;
+import org.wso2.carbon.module.swiftiso20022.validation.common.ValidationResult;
 
-import java.util.List;
 import java.util.Optional;
 
 /**
```

Comment:
Let's add a method comment to the `connect` method.

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/6/commits/8076e9b5033a9608b6839de38320ec994e01055d